test/components/DonationForm/DonationForm.spec.ts

```diff
@@ -1,29 +1,54 @@
-import { describe, it } from 'vitest';
-import { mount } from '@vue/test-utils';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { mount, VueWrapper } from '@vue/test-utils';
 import DonationForm from '@src/components/DonationForm/DonationForm.vue';
 import { DonationFormItems } from '@src/utils/FormItemsBuilder/DonationFormItems';
+import { Intervals } from '@src/utils/FormItemsBuilder/fields/Intervals';
+import { AddressTypes } from '@src/utils/FormItemsBuilder/fields/AddressTypes';
+import { PaymentMethods } from '@src/utils/FormItemsBuilder/fields/PaymentMethods';
 
 const formItems: DonationFormItems = {
-	addressType: [],
-	amounts: [ { value: '0', label: '€0', className: '' } ],
-	intervals: [],
-	paymentMethods: []
+	addressType: [ AddressTypes.NO, AddressTypes.EMAIL ],
+	amounts: [
+		{ value: '1', label: '€1', className: 'amount-1' },
+		{ value: '5', label: '€5', className: 'amount-5' }
+	],
+	intervals: [ Intervals.ONCE, Intervals.MONTHLY ],
+	paymentMethods: [ PaymentMethods.PAYPAL, PaymentMethods.CREDIT_CARD ]
 };
 
 describe( 'DonationForm.vue', () => {
-	it.todo( 'updates the interval when one is selected', () => {
-		const wrapper = mount( DonationForm, {
+	let wrapper: VueWrapper<any>;
+
+	beforeEach( () => {
+		wrapper = mount( DonationForm, {
 			props: {
 				formUrl: 'https://example.com',
 				customAmountPlaceholder: 'custom-amount-placeholder'
 			},
 			global: {
-				provide: { formItems: formItems }
+				provide: { formItems: formItems },
+				mocks: {
+					$translate: ( key: string ) => key
+				}
 			}
 		} );
+	} );
+
+	it( 'updates the interval when one is selected', async () => {
+		await wrapper.find( '.interval-1 input' ).trigger( 'change' );
+
+		expect( wrapper.find<HTMLInputElement>( 'input[name=interval]' ).element.value ).toBe( '1' );
```

Comment:
Selecting the input is just testing the SelectGroup inside the DonationForm. I'd rather have some other indicator of changed state, e.g. a SubmitValues component to really test that the changed state gets passed to the form.